app/models/manageiq/providers/redfish/physical_infra_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,38 @@
+require "server_sent_events"
+
+module ManageIQ::Providers::Redfish
+  class PhysicalInfraManager::EventCatcher::Runner \
+      < ManageIQ::Providers::BaseManager::EventCatcher::Runner
```

Comment:
Don't worry about the line-limit bot on this one, it is more readable on one-line